themes/fast-track-theme/lms/templates/courseware/accordion.html

```diff
@@ -22,31 +22,106 @@
 %>
 <a href="#${chapter['display_id']}-child" role="button" class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-expanded="false">
     <span class="group-heading ${active_class}" aria-label="${aria_label}">
-        <span class="icon fa fa-caret-right" aria-hidden="true"></span>
-        % if chapter['viewed']:
-            <i class="fa fa-circle"></i>
-        % else:
-            <i class="fa fa-circle-o"></i>
-        % endif
-        ${chapter['display_name']}
-        <p class="chapter-description">${chapter['display_description']}</p>
+        <div class="icon-container">
+            % if chapter['viewed']:
+                <i class="fa fa-check-circle"></i>
+            % endif
+        </div>
+        <span class="display-name">${chapter['display_name']}</span>
     </span>
 </a>
 <div class="chapter-content-container" id="${chapter['display_id']}-child" tabindex="-1" role="group" aria-label="${chapter['display_name']} submenu">
+## TODO: remove inline css when sidebar is finished!
```

Comment:
Yes, I see how ugly this is. 
Will be extracted and dried out when the sidebar is done.